_data/tagging-status.yml

```diff
@@ -77,26 +78,28 @@
  - name: enumerate
    type: package
    status: currently-incompatible
-   comments: See comments in [[1]](#ref1).
+   references: [1]
 
  - name: enumitem
    type: package
    status: partial-support
-   comments: See comments in [[1]](#ref1). The package itself can't be loaded right now. However, the tagging code supports (most of) the document-level  syntax of `enumitem`, so that one cae use, for example, `\begin{itemize}[...]` out of the box.
+   comments: The package itself can't be loaded right now. However, the tagging code supports (most of) the document-level  syntax of `enumitem`, so that one cae use, for example, `\begin{itemize}[...]` out of the box.
    tasks: Check which keys are not yet supported and what else is missing. 
    issues: [61]
    related-issues: [4]
    updated: 2024-07-05
-
+   references: [1]
+   
```

Comment:
cosmetic detail, but I would put reference directly after comments

Reply:
yes I added it there as it simplified conversion but I can move, I did think coming after updated looked a bit odd.

Reply:
fixed at ae5a8f4